.github/actions/install-dpcpp/action.yml

```
@@ -12,13 +12,24 @@ inputs:
     description: Path to DPC++
     type: string
     default: ~/dpcpp
+  GPU:
+    description: "Install for PVC or BMG gpu"
+    type: string
+  IGC:
+    description: "Use release of staging release"
+    type: string
+
 runs:
   using: "composite"
   steps:
     - name: Install DPCPP
       if: inputs.DPCPP_RELEASE == 'NIGHTLY'
       shell: bash
       run: |
+        if [[ "${{ inputs.GPU }}" != "BMG" ]]; then
+          if [[ "${{ inputs.IGC }}" != "staging" ]]; then
```

It's probably better to do a single line
```suggestion
        if [[ "${{ inputs.GPU }}" != "BMG" && "${{ inputs.IGC }}" != "staging" ]]; then
```